data-reconciliation-app/Makefile

```diff
@@ -88,4 +88,14 @@ deploy-mccf: ## 🚀 Deploy Managed CCF
 	@cd .. && $(MAKE) deploy-mccf
 
 deploy-ms-idp: ## 🔐 Create an Identity Provider
-	@cd .. && $(MAKE) deploy-ms-idp
\ No newline at end of file
+	@cd .. && $(MAKE) deploy-ms-idp
+
+publish-app-sandbox: build ## 🧪 Deploy the Data Reconciliation Application to a network
```

Comment:
I wonder if this command is better when we call it `publish-app-to-sandbox`?

Reply:
Done